CodeCoverage.runsettings

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<RunSettings>
+  <DataCollectionRunSettings>
+    <DataCollectors>
+      <DataCollector friendlyName="XPlat code coverage">
+        <Configuration>
+          <Format>json,cobertura</Format>          
+          <ExcludeByAttribute>Obsolete,GeneratedCodeAttribute,CompilerGeneratedAttribute</ExcludeByAttribute>
+          <SingleHit>false</SingleHit>
+          <UseSourceLink>true</UseSourceLink>
+          <IncludeTestAssembly>true</IncludeTestAssembly>
```

Comment:
Could you explain what these configs are for? Is there good docs on this stuff? I'm not very familiar with this :)

Reply:
I found this document that seems to be relevant: https://github.com/tonerdo/coverlet/blob/master/Documentation/VSTestIntegration.md